Custumize Typst displayMath

dongguaguaguagua opened the pull request:
Related issue(s):https://github.com/textext/textext/issues/417

Precise description of the changes proposed in the pull request:

In typst, the display math is marked as `$ Formula $`. 

This pull changes the build-in text if user is using Typst.

Short checklist:
- [x] Tested with Inkscape version: [1.3.2]
- [x] Tested on Linux
- [x] Tested on Windows
- [x] Tested on MacOS, Version:  [Ventura 13.5]

jcwinkler commented:
Thanks, @dongguaguaguagua I will do some additional checks under Win and Lin, then I will merge this. I will also port this into TexText 2.0.

dongguaguaguagua commented:
> Hi @dongguaguaguagua Could you please rebase your commit on the current state of the **develop** branch? Thanks!

Sorry about that. I've just added a `develop` branch to my fork and modified based on that. Should I make another pull request?
